app/src/main/java/fr/free/nrw/commons/bookmarks/locations/BookmarkLocationsDao.java

```diff
@@ -156,8 +157,11 @@ Place fromCursor(Cursor cursor) {
                 cursor.getString(cursor.getColumnIndex(Table.COLUMN_DESCRIPTION)),
                 location,
                 cursor.getString(cursor.getColumnIndex(Table.COLUMN_CATEGORY)),
-                builder.build()
+                builder.build(),
+                null,
+                null
         );
+        // TODO: add pic and destroyed to bookmark location dao
```

Comment:
Does this still need to be done?

Reply:
this needs to be done

Reply:
Actually after checking the code, I found out that this is not needed since if someone is bookmarked, it is always red. Being bookmarked has highest priority. But I want to keep the TODO, in case any problem occurs we will know where to look at.